aws_cloudtrail_rules/aws_manual_securtygroup_changes.py

```diff
@@ -0,0 +1,46 @@
+from panther_base_helpers import deep_get, pattern_match_list
+
+PROD_ACCOUNT_IDS = {'11111111111111', '112233445566'}
+SG_CHANGE_EVENTS = {
+    'CreateSecurityGroup': {
+        'fields': ['groupName', 'vpcId'],
+        'title': 'New security group [{groupName}] created by {actor}',
+    },
+    'AuthorizeSecurityGroupIngress': {
+        'fields': ['groupId'],
+        'title': 'User {actor} has updated security group [{groupId}]',
+    },
+    'AuthorizeSecurityGroupEgress': {
+        'fields': ['groupId'],
+        'title': 'User {actor} has updated security group [{groupId}]',
+    },
+}
+
+ALLOWED_USER_AGENTS = {
+    '* HashiCorp/?.0 Terraform/*',
+    # 'console.ec2.amazonaws.com',
+    # 'cloudformation.amazonaws.com',
+}
```

Comment:
Hmm usually not a fan of relying on user agent strings for whitelisting, since anyone can just set their user agent to whatever they want. Not sure if there is an different piece of information we could rely on, like user name or role.

Reply:
Perhaps a combination of allowed ip + user/role + useragent (decreases the likelihood someone has perfectly crafted _all_ pieces, but makes this more cumbersome)

Reply:
Role + UserAgent is pretty solid, I'll make the change